guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
+            click and drag the bar.
+          </para>
+
+          <para>It is not possible to individually adjust the size of panel 
+            columns.
+          </para>
+
+          <para><emphasis>Fields</emphasis></para>
+          <itemizedlist>
+            <listitem>
+              <para><guilabel>Title</guilabel> &mdash; Shows the title for the
+              highlighted (selected) lot in the <guilabel>Lots in This Account
+                </guilabel> panel. When a new lot is created by clicking the
+                <guilabel>New Lot</guilabel> button, it has a default title of
+                <emphasis>Lot n</emphasis> where n is a number starting at 0
+                (for each security account) and incrementing by 1. Deleting a
+                lot does not reset the next number to be used, but the title
+                can be changed as required.
+              </para>
+            </listitem>
+
+            <listitem>
+              <para><guilabel>Notes</guilabel> &mdash; Notes for the highlighted
+                lot. Users are free to enter anything they wish and the notes
+                can be changed as required.
+              </para>
+            </listitem>
+          </itemizedlist>
+          
+          <para><emphasis>Panels</emphasis></para>
+
+          <para>The order of the items in all the panels in this window
+            can be adjusted by clicking on the column headings.
+          </para>
+          
+          <itemizedlist>
+            <listitem>
+              <para><guilabel>Lots in This Account panel</guilabel></para>
+              <para>This panel shows lots for this security account.</para>
+              <para>If <guilabel>Show only open lots</guilabel> is checked, only
+                lots that have not been completely sold are shown.
+              </para>
+              <para><emphasis>Columns</emphasis></para>              
+              <itemizedlist>
+                <listitem><para><guilabel>Type</guilabel> &mdash; Always
+                  blank for security lots. This window is also used for business
+                  lots (which link payments to invoices) when coming from an
+                  Accounts Payable or Receivable account and in that case,
+                  <guilabel>Type</guilabel> will be <emphasis>I</emphasis> for
+                  invoice.</para>
+                </listitem>
+
+                <listitem>
+                  <para><guilabel>Opened</guilabel> &mdash; <emphasis>Open
+                    </emphasis> if there are no splits in the lot, otherwise the
+                    date of the earliest split in this lot.</para>
+                </listitem>
+
+                <listitem><para><guilabel>Closed</guilabel> &mdash; <emphasis>
+                  Open</emphasis> if the lot is not fully sold, otherwise the
+                  date of the last sell split.</para>
+
+                  <note>
+                    <para>Due to <ulink
```

Comment:
Why not? It is what it is. This could save people hours investigating to see if they should raise a bug. I'll happily update the documentation if it's fixed. Would removing the word 'bug' be sufficient?